test/models/EmailTemplateSpec.scala

```diff
@@ -133,14 +133,14 @@ class EmailTemplateSpec extends SpecBase {
         ) mustBe expectedEmailTemplate
       }
 
-      "file role is StandingAuthority" in new Setup {
```

Comment:
Could we create a separate CDSCashAccount test instead of modifying the StandingAuthority unit test? StandingAuthority is still being used, so we need the unit test

Reply:
apologies, I must have overridden the change on top of it.  I have included it back